### DISCUSSION TITLE
Can't import beatmaps on mac

### DISCUSSION THREAD

**Discussion by gabrieldabbah** [ppy/osu#17518] [Q&A] [answered]
I have an osu stable folder with beatmaps + skins, etc.

I can't click on the "select directory" button when trying to import beatmaps. I tried changing the lazer location, the stable location, etc.

I can't import songs by dragging .osu to my lazer window.

**Comment by peppy** [comment, marked as answer]
lazer should fine the stable folder automatically if it's in your home directory. Is it in a different location? If so, please try to move it back to `~/.osu`.

**Reply by gabrieldabbah** [reply]
I believe lazer is already on the home folder and so is stable. The home folder is the folder with your mac's username, correct? Finder's toolbar home opens my username folder. Both lazer and stable are there. I still can't select the directory. I'll include a screenshot.
<img width="645" alt="Screen Shot 2022-03-28 at 23 13 05" src="https://user-images.githubusercontent.com/63445917/160525725-288b2d71-877d-41ea-92dd-9fc07a8d34f4.png">
.

**Reply by peppy** [reply]
Where is the stable install from? Were you not previously using the official wine version on mac?

**Reply by gabrieldabbah** [reply]
No. Before I Changed to mac, I saved my beatmaps folder, skins folders, and collections.db.
I tried importing creating an "osu!" folder holding the previously mentioned folders. It didn't work. So I asked a friend to send me a zip of their osu game folder (yes, they knew it contained sensitive info, but I wouldn't do anything with it). I tried installing osu stable through the guide. It didn't work correctly. It kept crashing and freezing my computer (MacBook Pro 2021).

**Reply by peppy** [reply]
You need to name the folder `.osu` not `osu!`.

Open terminal and run

`mv osu\! .osu`

**Reply by gabrieldabbah** [reply]
That worked. Thx ppy
